### PR TITLE
responsive_navbar

### PR DESCRIPTION
**Pull request by MuhammadFahimulHuq**
- [x] toggle hamburger.
- [x] used prettier for formatting.

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,7 +1,13 @@
-import React from "react";
+import React, { useState } from "react";
 import { GymName } from "../constants";
 
 const Navbar = () => {
+  // set the default dropmenu as false because we want to hide it in smaller screen
+  const [dropMenu, setDropMenu] = useState(false);
+  // function for toggling drop menu
+  const toggleDropMenu = () => {
+    setDropMenu(!dropMenu);
```
Comment (Min-Ho-Lim) [inline src/components/Navbar.tsx]: This should do the job, but this is not a good practice in the future when you are dealing with `useState` in React.

`useState` is an asynchronous hook and it doesn't change the state immediately, it has to wait for the component to re-render. If we have another places to dealing with `dropMenu`, we might suffer with race conditions.

Use prevState instead. 

```suggestion
    setDropMenu(prevState => !prevState);
```

If you want to learn more about this, please you can read this.
https://www.delftstack.com/howto/react/react-setstate-prevstate/

Comment (Min-Ho-Lim) [reply src/components/Navbar.tsx]: - [x] Change the code to use `prevState`

**Review by Min-Ho-Lim — CHANGES_REQUESTED** [review]

File: src/components/Navbar.tsx (hunk 1 above)
Comment (MuhammadFahimulHuq) [reply src/components/Navbar.tsx]: Thank you for your inputs. 
- [x] Change the code to use ```prevState```

**Review by Min-Ho-Lim — APPROVED** [review]
Tested locally. Thanks for your contribution. Approved.

**Pull request merged**